src/libsync/configfile.cpp

```diff
@@ -363,6 +363,17 @@ QString ConfigFile::configPath() const
     return dir;
 }
 
+QString ConfigFile::localConfigFile() const
+{
+    const QString settingspath = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
+    if(!QDir(settingspath).exists()) {
+        QDir().mkdir(settingspath);
+    }
+    QString filename = settingspath + QLatin1String("/") + QLatin1String("settings.cfg");
```

Comment:
Similarly to the `localConfigFile` not being expressive of what this file is for, `settings.cfg` has the same issue